src/components/table/TableColumn.vue

```diff
@@ -27,6 +27,10 @@ export default {
         subheading: [String, Number],
         customSort: Function,
         sticky: Boolean,
+        headerSelectable: {
+            type: Boolean,
+            default: true
```

Comment:
No It's a breaking, so you have to set to false at the moment

Reply:
Seems OK, since the prop determine if the header is selectable. `true` is the current behavior.

Reply:
The style binding is inverted (`'is-unselectable': !column.headerSelectable`), so if default is True, no style is added. This should not be breaking?